src/demux.js

```diff
@@ -277,6 +275,18 @@ class DemuxSink extends Writable {
 
         const { statsSessionId, type, data } = request;
```

Comment:
Here we could use the utility method that I mentioned [earlier](https://github.com/jitsi/rtcstats-server/pull/99/files#r1045622757).